packages/cache/tests/lib/CacheMiddleware.test.ts

```diff
@@ -0,0 +1,70 @@
+import { MapProvider } from '@joshdb/map';
+import { Method, MiddlewareStore } from '@joshdb/middleware';
+import { CacheMiddleware } from '../../src';
+
+describe('CacheMiddleware', () => {
```

Comment:
Just a reminder we want to hit at least 80% coverage

Reply:
I'm going for 100% dw